partcad/src/partcad/ai_feature_file.py

```
@@ -0,0 +1,92 @@
+    def __init__(self, url: str):
+        self.url = url
+        self.is_text = False
+        self.is_image = False
+        self.is_video = False
+        self.is_pdf = False
+        self.suffix = ".txt"
+        self.filename = None
+
+        # Switch to lowercase to normalize the file extension in the URL
+        lower_url = url.lower()
+        if lower_url.startswith("http://") or lower_url.startswith("https://"):
```

Ruff (0.8.2)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)